Frameworks/Core/ERExtensions/Sources/er/extensions/foundation/ERXSimpleTemplateParser.java

```diff
@@ -35,7 +35,7 @@
     public static final String DEFAULT_DELIMITER = "@@";
 
     /** logging support */
-    private static final Logger log = LoggerFactory.getLogger(ERXSimpleTemplateParser.class);
+    private static final Logger LOG = LoggerFactory.getLogger(ERXSimpleTemplateParser.class);
```

Comment:
I think we should keep the character case of that variable. Surely it is quite common to make static variables uppercase but in the case of the _log_ variable we have it lowercase in all Wonder source.